plugins/inputs/lotus/lotus.go

```diff
@@ -132,6 +136,15 @@ func (l *lotus) Start(acc telegraf.Accumulator) error {
 	}
 	l.api = nodeAPI
 
+	nodePeerID, err := l.api.ID(context.Background())
+	if err != nil {
+		return err
+	}
```

Comment:
Push the nodePeerID getting into `recordLotusInfoPoints`

Reply:
:+1: done